### PR TITLE
Create chocolate dessert drink

### PR DESCRIPTION
**Pull request by s194317**
Added hot chocolate recipe.

### CODE REVIEW

**Comment by jakobtkn** [comment]
I believe the title should be "Create chocolate dessert drink"

File: desserts/chokolate desert drink [hunk 1]
```diff
@@ -0,0 +1,6 @@
+40g Dark chocolate (70%)
+30g whipped cream
+2 tbsp pepper
```
Comment (jakobtkn) [inline desserts/chokolate desert drink:3]: This does not sound too nice

File: desserts/chokolate desert drink [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+2dL water
+
+Smelt the chokolat, whip the cream, and mix it together.
```
Comment (jakobtkn) [inline desserts/chokolate desert drink:6]: I would go with chocolate

**Review by jakobtkn — CHANGES_REQUESTED** [review]

File: desserts/chokolate desert drink (hunk 2 above)
Comment (Lachrynier) [inline desserts/chokolate desert drink:6]: i would suggest moving each instruction onto a separate line

File: desserts/chokolate desert drink (hunk 1 above)
Comment (s194317) [reply desserts/chokolate desert drink:3]: Hmm okay, if you don't like pepper, I'll remove it.

> This does not sound too nice

**Review by budziszpiotr — APPROVED** [review]
sounds delicious!

**Pull request merged**